origin-bridge/src/utils/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const Web3 = require('web3')
+const web3 = new Web3(process.env.PROVIDER_URL || 'http://localhost:8545')
```

Comment:
Same comment as above here. We should be able to use directly Web3, without instantiating an object.